yt_dlp/extractor/youtube.py

```diff
@@ -686,6 +698,13 @@ def _extract_time_text(self, renderer, *path_list):
         timestamp = None
         if isinstance(dt, datetime.datetime):
             timestamp = calendar.timegm(dt.timetuple())
+
+        if timestamp is None:
+            timestamp = (
+                unified_timestamp(text) or unified_timestamp(
+                    self._search_regex(
+                        (r'(?:.+|^)(?:streaming|live|ed)(?:\s*on)?\s*(.+)', r'\w+[\s,\.-]*\w+[\s,\.-]*20\d{2}'), text.lower(), 'time text', default=None)))
```

Comment:
the latter regex here by @MinePlayersPE is probably a bit overkill

currently I am assuming the language is always en_US, otherwise for other english dialects such as en_GB we'd need to set `day_first=False` in `unified_timestamp`